tools/Microsoft.StandardsPolice/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "version": "1.0.0-*",
 
+  "compilationOptions": {
```

Comment:
Hmmmm, @pakrym are we still strong naming meta assemblies? @Eilon we need to figure out if this will affect everyone doing ICompileModule(s)